handbook/engineering/README.md

```diff
@@ -669,25 +672,28 @@ After it's in a release formally, the bug should be treated like any other piece
 #### Fast track for Fleeties
 Fleeties do not have to wait for QA to reproduce the bug. If you're confident it's reproducible, it's a bug, and the reproduction steps are well-documented, it can be moved directly to the reproduced state.
 
-#### Release testing
-When a release is in testing, QA should use the the Slack channel #help-qa to keep everyone aware of issues found. All bugs found should be reported in the channel after creating the bug first.
+### Release testing
+When a release is in testing, QA should use the Slack channel #help-qa to keep everyone aware of issues found. All bugs found should be reported in the channel after creating the bug first.
 
-In the #help-qa channel, product may decide whether the bug is a release blocker. When a release-blocking bug is found, product is responsible for communicating the delay to company stakeholders. 
+In the #help-qa channel, product may decide whether the bug is a critical or release-blocking. When a critical bug is found, product is responsible for following the [critical bug notification process](https://fleetdm.com/docs/contributing/releasing-fleet#critical-bug-notification-process) below. When a release-blocking bug is found, product is responsible for communicating any delays to company stakeholders. 
 
-Release blockers include:
-1. Critical bugs (defined below)
-2. New functionality that impacts previous stable functionality
-3. Incomplete features as defined or implied in the specs
+Release blockers must be fixed before a release can be cut. Non-release-blocking bugs may be addressed during a subsequent release per the standard [bug process](https://fleetdm.com/docs/contributing/releasing-fleet#bug-process). 
 
-Release blockers must be fixed before a release can be cut. Non-release-blocking bugs may be addressed during a subsequent release per the standard bug process (defined above).
+### Release-blocking bugs
+A release-blocking bug is identified by the `~release blocker` label. It becomes a release-blocking bug if:
+* New functionality that impacts previous stable functionality
+* Incomplete features as defined or implied in the specs
+* It does not meet the criteria of a critical bug
+* We will delay the release to ensure a fix if necessary
```

Comment:
This seems like it doesn't belong in this list. This is the definition of a release-blocking bug, so it can't be something that determines the criteria for being a release-blocking bug. It's circular there.

Reply:
I see. So what I'm trying to do is create more discipline/clarity on the product side: what criteria do we need in order to declare a bug critical or release-blocking. It sounds like you're saying PM just decides, I'm trying to provide more rails around it. 

In the previous version of this doc, it had said: a critical bug is one that meetings criteria a b c (which is well-preserved in this version). But then the release-blocking bug is any critical bug on an existing released feature OR some new feature in this release is incomplete/not to spec. And release-blocking bugs do not need warning to users since it hasn't actually been released. 

I am open to not laying out those definitions here, but I think they provide more clarity in situations when a PM and EM disagree.

Reply:
I see. So what I'm trying to do is create more discipline/clarity on the product side: what criteria do we need in order to declare a bug critical or release-blocking. It sounds like you're saying PM just decides, I'm trying to provide more rails around it. 

In the previous version of this doc, it had said: a critical bug is one that meetings criteria a b c (which is well-preserved in this version). But then the release-blocking bug is any critical bug on an existing released feature OR some new feature in this release is incomplete/not to spec. And release-blocking bugs do not need warning to users since it hasn't actually been released. 

I am open to not laying out those definitions here, but I think they provide more clarity in situations when a PM and EM disagree.

Reply:
That makes sense. We can remove this definition entirely if we remove the concept of a release-blocking bug. Any unreleased bug would be treated as release-blocking by default, and any critical bug would receive a patch. Other bugs on the board are prioritized by ordering the "Ready" column. If there are especially important bugs, they should be prioritized that way, and by commenting on the bug. 

Several folks have mentioned adding some priority labels to bugs, but I think the danger there is bugs labeled low priority will never get looked at. I'm also inclined to err on the side of less labels. 

Will revise and retag you for review.